lrf_to_grid_node:
  ros__parameters:
    lrf_to_grid:
      topic_name:
        map: "/lrf/map"
        storage_map: "/lrf/storage_map"
        laser: "/scan"
        cloud: "/glim_ros/points"
      tf_frame:
        map_frame: "map"
        robot_frame: "base_link"
      add_cell: true
      storage_map:
        publish_rate: 1.0 #[s]
        que_size: 10
      filter:
        voxelgrid_size: 0.1
        z_max: 0.5 #1.2
        z_min: -0.5 #0.5
      grid_map_info:
        resolution: 0.2 #[m] def 0.05
        # origin: [-5.0,-5.0,0.0]
        width: 20.0 #[m] 25.0
        height: 20.0 #[m] 25.0
